README.md

```diff
@@ -101,12 +89,56 @@ patch(vnode, newVnode); // Snabbdom efficiently updates the old view to the new
 patch(newVnode, null)
 ```
 
-## Examples
+## More examples
 
 * [Animated reordering of elements](http://snabbdom.github.io/snabbdom/examples/reorder-animation/)
 * [Hero transitions](http://snabbdom.github.io/snabbdom/examples/hero/)
 * [SVG Carousel](http://snabbdom.github.io/snabbdom/examples/carousel-svg/)
 
+* * *
+
+## Table of contents
+
+* [Core documentation](#core-documentation)
+  * [`snabbdom.init`](#snabbdominit)
+  * [`patch`](#patch)
+  * [`snabbdom/h`](#snabbdomh)
+  * [`snabbdom/tovnode`](#snabbdomtovnode)
+  * [Hooks](#hooks)
+    * [Overview](#overview)
+    * [Usage](#usage)
+    * [The `init` hook](#the-init-hook)
+    * [The `insert` hook](#the-insert-hook)
+    * [The `remove` hook](#the-remove-hook)
+    * [The `destroy` hook](#the-destroy-hook)
+  * [Creating modules](#creating-modules)
+* [Modules documentation](#modules-documentation)
+  * [The class module](#the-class-module)
+  * [The props module](#the-props-module)
+  * [The attributes module](#the-attributes-module)
+  * [The dataset module](#the-dataset-module)
+  * [The style module](#the-style-module)
+    * [Custom properties (CSS variables)](#custom-properties-css-variables)
+    * [Delayed properties](#delayed-properties)
+    * [Set properties on `remove`](#set-properties-on-remove)
+    * [Set properties on `destroy`](#set-properties-on-destroy)
+  * [Eventlisteners module](#eventlisteners-module)
+* [SVG](#svg)
+  * [Classes in SVG Elements](#classes-in-svg-elements)
+* [Thunks](#thunks)
+* [Virtual Node](#virtual-node)
+  * [sel : String](#sel--string)
+  * [data : Object](#data--object)
+  * [children : Array<vnode>](#children--arrayvnode)
+  * [text : string](#text--string)
+  * [elm : Element](#elm--element)
+  * [key : string | number](#key--string--number)
+* [Structuring applications](#structuring-applications)
+* [Common errors](#common-errors)
+* [Maintenance policy](#maintenance-policy)
+  * [Pull requests](#pull-requests)
+  * [Releases](#releases)
```

Comment:
Full-depth table of contents, because I feel it's useful for navigation and as an overview of the available documentation.